packages/graphql-language-service-types/package.json

```diff
@@ -14,21 +14,23 @@
   },
   "license": "MIT",
   "files": [
-    "dist"
+    "dist",
+    "esm"
   ],
   "keywords": [
     "graphql"
   ],
   "main": "dist/index.js",
-  "typings": "src/index",
+  "module": "esm/index.js",
+  "typings": "src/index.ts",
   "scripts": {
     "build": "yarn run build-ts && yarn run build-flow",
     "build-ts": "tsc",
     "build-flow": "node ../../resources/buildFlow.js",
     "prepublish": "node ../../resources/prepublish.js"
   },
   "peerDependencies": {
-    "graphql": "^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0"
+    "graphql": ">= ^0.10.0 <= ^14.0.0"
```

Comment:
Invalid semver

Reply:
Also this differs from the previous definitions in that it has extra spaces. Best to try and keep the specs as similar to each other as possible so that lock files don't get too verbose.

Reply:
yes, it might be helpful to use this shorter semver once we need to update it, say for 15.0.0 releases which may come soon for the alpha.1